src/stores/AppStore.js

```diff
@@ -184,6 +195,8 @@ class AppStore extends EventEmitter {
                 this.emit('customEventsChange');
                 break;
             case 'LOAD_SCHEDULE':
+                // If the user already had schedules, load up four schedules
+                this.scheduleNames = ['Schedule 1', 'Schedule 2', 'Schedule 3', 'Schedule 4'];
```

Comment:
I know the backend isn't ready yet, but how will this work when a user has saved named schedules?

Reply:
If we use the current schema and slightly adjust it so that it handles schedule names, we can just set `this.scheduleNames` equal to the user's saved `scheduleNames` array.

If we use the other schema, we can iterate through the user's schedules and put all the names into an array.